modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,131 @@
+* `admin` - By default, no users have this role. The default admin role provides full access to all resources in your Redpanda installation.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs can contain allow permissions, deny permissions, or a combination of both. As a result, users may at times have conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+== Managing users and roles
+
+Redpanda allows you to manage your RBAC configurations through both RPK and the web console. Examples of each are given here.
+
+=== Create a role
+
+Creating a new role is a two-step process. First you define the role, giving it a unique and descriptive name. Second, assign that role as the allowed or denied principal to one or more ACLs. This defines the permissions associated with the role.
+
+Using RPK, administrators create a new role by issuing the command:
+
+[,bash]
+----
+rpk role create [ROLE_NAME]
+----
+
+Once the role is created, administrators assign the role as the allowed principal for one or more ACLs. You can modify existing ACLs or create new ones. For example:
+
+[,bash]
+----
+rpk acl create ... --allow-principal Role:[ROLE_NAME]
+----
+
+=== Delete a role
+
+When a role is deleted, Redpanda carries out the following actions automatically:
+
+- All role ACLs are deleted.
+- All users are unassigned from role.
+
+Using RPK, administrations delete a role by issuing the command:
+
+[,bash]
+----
+rpk role delete [ROLE_NAME]
+----
+
+=== Assign a role
+
+Administrators may assign a role to any principal created in their Redpanda instance. A principal is composed of two parts: a type and a name. Redpanda currently supports the `User` type only for principals. All principals, therefore, will have the format of `User:[name]`
+
+Using RPK, administrations assign a role to a principal by issuing the command:
+
+[,bash]
+----
+rpk role assign [ROLE_NAME] [PRINCIPAL]
+----
+
+=== Unassign a role
+
+Administrators may remove a role from a principal without deleting the role. A principal is composed of two parts: a type and a name. Redpanda currently supports the `User` type only for principals. All principals, therefore, will have the format of `User:[name]`
+
+Using RPK, administrators unassign a role from a principal issuing the command:
+
+[,bash]
+----
+rpk role unassign [ROLE_NAME] [PRINCIPAL]
+----
+
+=== Edit a Role
+
+After a role is created, administrators may still add additional ACLs to the role. Redpanda also allows renaming a role.
+
+Using RPK, Administrators may rename a role by issuing the command:
+
+[,bash]
+----
+rpk role edit [ROLE_NAME] --rename [NEW_NAME]
+----
```

Comment:
This functionality (role renaming) has been abandoned.

Reply:
Removed the reference.